master/buildbot/worker/openstack.py

```diff
@@ -88,6 +89,8 @@ def __init__(self, name, password,
             self.novaclient = self._constructClient(
                 client_version, os_username, os_password, os_tenant_name,
                 os_auth_url)
+            if region is not None:
+                self.novaclient.client.region = region
```

Comment:
The [docs](https://docs.openstack.org/python-novaclient/latest/reference/api/novaclient.v2.client.html) show the Client supporting a `region_name` parameter.